cmd/apiserver/server.go

```diff
@@ -60,12 +57,69 @@ func (s *server) createCalculationBulk(c *gin.Context) {
 	}
 
 	if err := s.client.Create(s.ctx, bulk); err != nil {
-		responseError(c, "couldn't create calculation", err)
+		responseError(c, "couldn't create calculation bulk", err)
 	} else {
 		c.JSON(http.StatusOK, gin.H{"data": bulk})
 	}
 }
 
+func (s *server) createWorkerPool(c *gin.Context) {
+	workerPoolName := c.Query("name")
+
+	workerPool := &workersv1.WorkerPool{}
+	workerPool.Name = fmt.Sprintf("workerpool-%s", workerPoolName)
+
+	s.logger.Infof("Creating the workerpool %s...", workerPool.Name)
+
+	if err := s.client.Create(s.ctx, workerPool); err != nil {
+		responseError(c, "couldn't create calculation bulk", err)
+	} else {
+		c.JSON(http.StatusOK, gin.H{"data": workerPool})
```

Comment:
Well, I dont know what changed since last time... What ways do we want the user to create the workerpool? Just to create empty object with a name + namespace?

Reply:
Yea but this is the response. You should get the created workerpool and send it back to user. Otherwise, you are sending an empty object.

Reply:
Dont we do the same thing with calculation bulk creation? I understand what you mean, but dont know how to do it.